app/views/contexts/summary/group.py

```diff
@@ -107,7 +113,18 @@ def _build_blocks_and_links(
                 if parent_list_collector_block_id not in routing_path_block_ids:
                     continue
 
-                list_collector_block = ListCollectorBlock(
+                list_collector_block_class: Type[
+                    ListCollectorBlock | ListCollectorContentBlock
```

Comment:
type hint not needed - mypy will infer it from the classes

Reply:
"ListCollectorBaseBlock" has no attribute "list_summary_element" when removed as mypy thinks we are using parent class and the method is not in that class.